hw4/tests/test_hw4_t03.py

```diff
@@ -0,0 +1,22 @@
+from hw4.hw4_t03 import my_precious_logger
+
+
+def test_stdout_simple_text(capsys):
+    my_precious_logger("basically, this is any text")
+    out, err = capsys.readouterr()
+    assert out == "basically, this is any text\n"
+    assert err == ""
+
+
+def test_stdout_mixed_text(capsys):
+    my_precious_logger("please raise error now")
+    out, err = capsys.readouterr()
+    assert out == "please raise error now\n"
+    assert err == ""
+
+
+def test_stderr_mixed_text(capsys):
```

Comment:
Please do a simple experement.  

Ask someone to describe the function under the test by these test names.  
```
test_stderr_mixed_text
test_stdout_mixed_text
test_stdout_simple_text
```